ompi/runtime/ompi_spc.c

```diff
@@ -0,0 +1,281 @@
+#include "ompi_spc.h"
```

Comment:
Am I supposed to add copyright information to the top of the files that I added?

Reply:
Yes. On files where the UTK copyright was not present you should add it. On new files you should only add UTK copyright.